CodeStarterCamp_Week1/makePepero.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  makePepero.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by daekyo on 2023/09/26.
+//
+
+import Foundation
+
+func makePeperoHand(handCount: Int) {
+    for _ in 1...handCount {
+        print(" | |")
+    }
+}
+
+func makePeperoSingleBody(bodyStyle: String, topingStyle: String = " ") -> String {
+    return topingStyle + bodyStyle + topingStyle
+}
+
+func makePeperoBody(singleBody: String, bodyCount: Int) {
+    for _ in 1...bodyCount {
+        print(singleBody)
+    }
+}
+
+func makePepero(bodyCount: Int, bodyStyle: String, handCount: Int, topingStyle: String = " ") {
+    print("<정보>")
+    print("길이: \(bodyCount)")
+    print("몸통: \(bodyStyle)")
+    print("토핑: \(topingStyle)")
+    print("막대길이: \(handCount)")
+    print("")
```

Comment:
문자열 보간법을 잘 활용해주셨네요 👍
여러 줄 문자열과 특수문자를 활용해보셔도 좋을 것 같습니다. 💭
```suggestion
    let peperoInformation = """
    <정보>
    길이: \(bodyCount)
    몸통: \(bodyStyle)
    토핑: \(topingStyle)
    막대길이: \(handCount)\n
    """
    print(peperoInformation)
```